src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/MetadataReferenceItemHandler.cs

```diff
@@ -11,11 +11,16 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices.Handlers
     /// </summary>
     [Export(typeof(ILanguageServiceCommandLineHandler))]
     [AppliesTo(ProjectCapability.CSharpOrVisualBasicLanguageService)]
-    internal class MetadataReferenceFilesLanguageServiceItemHandler : ILanguageServiceCommandLineHandler
+    internal class MetadataReferenceItemHandler : ILanguageServiceCommandLineHandler
     {
+        private readonly UnconfiguredProject _unconfiguredProject;
+
         [ImportingConstructor]
-        public MetadataReferenceFilesLanguageServiceItemHandler(UnconfiguredProject project)
+        public MetadataReferenceItemHandler(UnconfiguredProject project)
         {
+            Requires.NotNull(project, nameof(project));
```

Comment:
I guess this does not hurt but this call just doesn't do anything. We should bigger sweep to remove all of this.